src/main/java/me/karakelley/tictactoe/GameConfiguration.java

```diff
@@ -41,7 +41,7 @@ public Player getPlayer1() {
   }
 
   public void chooseOpponent(String choice) {
-    if (choice != "1") {
+    if (!choice.equals("1")) {
```

Comment:
Magic strings are not your friends :). Extracting them into a constant is a better practice

Reply:
I'm not sure what you mean by magic strings. Can you clarify?